pybind11 was checked out during src_unpack to prevent sandbox violations
diff --git a/setup.py b/setup.py
index e84f7fd..c34dd62 100644
--- a/setup.py
+++ b/setup.py
@@ -113,8 +113,10 @@ class CMakeBuild(build_ext):
 
     def build_extension(self, ext):
         extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
         cmake_args = [
             "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=" + str(extdir),
+            "-DCMAKE_LIBRARY_ARCHITECTURE=" + str(os.environ["CMAKE_LIBRARY_ARCHITECTURE"]),
+            "-DFETCHCONTENT_SOURCE_DIR_PYBIND11-SRC=" + str(os.environ["PYBIND11_SRC_DIR"]),
             "-DPYTHON_EXECUTABLE=" + sys.executable,
         ]
 
